components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -49,195 +49,197 @@
+                    for (int j = 0; j < eventStreamAttributeListSize; j++) {
+                        Attribute.Type attributeType = executionPlanRuntime.getStreamDefinitionMap().get(inputStreamId)
+                                .getAttributeList().get(j).getType();
+                        switch (attributeType) {
+                            case STRING:
+                                object[j] = eventStreamAttributeArray[j];
+                                break;
+                            case INT:
+                                object[j] = Integer.parseInt(eventStreamAttributeArray[j]);
+                                break;
+                            case LONG:
+                                object[j] = Long.parseLong(eventStreamAttributeArray[j]);
+                                break;
+                            case FLOAT:
+                                object[j] = Float.parseFloat(eventStreamAttributeArray[j]);
+                                break;
+                            case DOUBLE:
+                                object[j] = Double.parseDouble(eventStreamAttributeArray[j]);
+                                break;
+                            case BOOL:
+                                object[j] = Boolean.parseBoolean(eventStreamAttributeArray[j]);
+                                break;
+                            case OBJECT:
+                                object[j] = eventStreamAttributeArray[j];
+                                break;
+                        }
+                    }
+                    //send events
+                    for (String key : inputHandlerMap.keySet()) {
+                        if (key.equals(inputStreamId)) {
+                            inputHandlerMap.get(key).send((startSetTime +
+                                    (System.currentTimeMillis() -
+                                            startSystemTime)), object);
+                        }
+                    }
+                } else if (delayPatternMatcher.find()) {
+                    Thread.sleep(Long.parseLong(delayPatternMatcher.group(2)));
+                } else {
+                    if (!inputStreamEventArray[i].equals("")) {
+                        errMsg = "Error in event \"  " +
+                                inputStreamEventArray[i] +
+                                "\n\"." +
+                                " Expected format: &lt;eventStreamName&gt;=[&lt;attribute1&gt;,&lt;attribute2&gt;]";
+                        throw new Exception(errMsg);
+                    }
+                }
+            }
+            Thread.sleep(500);
+            return map;
+        } catch (Throwable e) {
+            //catch throwable since there could be execution plan validation exceptions
+            errMsg = "Error occurred while processing. " + e.getMessage();
+            log.error(errMsg, e);
+            throw new Exception(errMsg, e);
```

Comment:
@prabathabey 
Thank you for pointing out. We have not created any component specific exceptions as this is only a ui component. As for now, Siddhi try it ui component escalates the error message to front UI (directly called by a jsp file). So IMHO, I believe the above approach will be okay. Please correct me if I am wrong.